oat-sa/tao-core
# Feature/tr 3033/login form rtl

Related to https://oat-sa.atlassian.net/browse/TR-3033

TODO: 
- [x] apply https://github.com/oat-sa/tao-core-ui-fe/pull/393

Example for another controller: https://github.com/oat-sa/extension-tao-testqti/blob/develop/views/js/controller/runner/runner.js#L176

How to test:
- in `config\generis.conf.php`, set `define('DEFAULT_ANONYMOUS_INTERFACE_LANG','ar-arb');`
- open main (login) page and check it's rendered correctly
